PTN/parse.py

```diff
@@ -98,6 +98,9 @@ def parse(self, name):
             raw = raw[self.start:self.end].split('(')[0]
 
         clean = re.sub('^ -', '', raw)
+        clean = re.sub('www.*.* ', ' ', clean)
```

Comment:
What about movies/shows that start with www?

Why twice regex of .* ?

What you replace to spa e and not just empty string?